docs/plugins/integrations.asciidoc

```diff
@@ -1,17 +1,13 @@
 [[plugin-integrations]]
-== Plugin integrations
+== Integration plugins
 
-Plugin integrations combine related plugins--inputs, outputs, and sometimes
+Integration plugins combine related plugins--inputs, outputs, and sometimes
 filters and codecs--into one package. 
 
 |=======================================================================
-| Plugin Integration | Description | Github repository
+| Integration Plugins | Description | Github repository
 | <<plugins-integrations-kafka,kafka>> | Plugins for use with the Kafka distributed streaming platform. | https://github.com/logstash-plugins/logstash-integration-kafka[logstash-integration-kafka]
 | <<plugins-integrations-rabbitmq,rabbitmq>> | Plugins for processing events to or from a RabbitMQ broker. | https://github.com/logstash-plugins/logstash-integration-rabbitmq[logstash-integration-rabbitmq]
 |=======================================================================
 
-:edit_url: https://github.com/logstash-plugins/logstash-integration-kafka/edit/master/docs/index.asciidoc
-include::integrations/kafka.asciidoc[]
-
-:edit_url: https://github.com/logstash-plugins/logstash-integration-rabbitmq/edit/master/docs/index.asciidoc
-include::integrations/rabbitmq.asciidoc[]
+:edit_url: https://github.com/elastic/logstash-docs/edit/master/docs/plugins/integrations.asciidoc[]
```

Comment:
Should stay as is.  This pulls in the asciidoc files for kafka and rabbitmq, and correctly sets the edit url overrides.

Reply:
My reason for this edit is that clicking the "edit" button in the [integrations list page](https://www.elastic.co/guide/en/logstash/7.5/plugin-integrations.html) jumps into rabbitmq  docs, which is confusing.

Reply:
You have uncovered a weird behavior change from our conversion to asciidoctor. @nik9000 has discovered that html5 fixes the edit_me URL issue and has [added a test](https://github.com/elastic/docs/pull/1509).